src/views/settings/Terminal/Endpoint/EndpointList.vue

```diff
@@ -6,6 +6,7 @@
       :header-actions="headerActions"
       :table-config="tableConfig"
       :create-drawer="createDrawer"
+      :resource="$t('Endpoint')"
     />
   </div>
 </template>
```

Comment:
The only difference between the updated version (`@@6,7 +@@` line) is an extra space added after `:` before "resource". This change does not affect function or variable definitions so it's minor and should be ignored. The rest of the code doesn't show any major changes that may require maintenance or updating.

If you want to improve performance, consider using single quotes instead of double quotes for the template strings in HTML to avoid issues with cross-platform rendering and ensure consistency across different environments. However, this doesn't modify existing functionality but serves more as a stylistic guideline rather than fixing actual bugs.